Emma/emma_vis_libs/helper.py

```diff
@@ -48,23 +48,16 @@ def getLastModFileOrPrompt(subStringIdentifier: str, inOutPath: str, quiet: bool
 
     # check if there is only one file which can be analyzed
     numberOfFiles = 0
-    for file in lastModifiedFiles:
-        if subStringIdentifier in file:
-            fileToUse = file
-            numberOfFiles += 1
-        if numberOfFiles > 1:
-            break
-    if numberOfFiles == 1:
-        return fileToUse
-
+    lastModifiedFiles = sorted(lastModifiedFiles)
     # Get last modified file (we NOT ONLY need this for the quiet mode)
     # Backwards iterate over file list (so newest file will be first)
-
-    for i in range(len(lastModifiedFiles) - 1, -1, -1):
+    for i, file in enumerate(lastModifiedFiles):
```

Comment:
I think this is wrong. We have to *backwards* iterate (see also comment above).

Reply:
Have you tested it with multiple reports as well?
Btw: Are there unit tests for this function (`getLastModFileOrPrompt`)?